internal/pkg/cli/env_init.go

```diff
@@ -532,15 +532,16 @@ If you proceed without at least two public subnets, you will not be able to depl
 			IsPublic: false,
 		})
 		if err != nil {
-			if err == selector.ErrSubnetsNotFound {
-				log.Errorf(`No existing private subnets were found in VPC %s. You can either:
-- Create new private subnets and then import them.
-- Use the default Copilot environment configuration.
+			if errors.Is(err, selector.ErrSubnetsNotFound) {
+				log.Warningf(`No existing private subnets were found in VPC %s.
+If you proceed without at least two private subnets, you may not be able to access 
+all Copilot features with this environment in the future.
```

Comment:
nit: maybe this to be more specific?
```suggestion
				log.Warningf(`No existing private subnets were found in VPC %s.
If you proceed without at least two private subnets, you won't be able 
to place services in private subnets.
```

Reply:
🤔  Seems like stating the obvious, maybe?
Maybe "If you proceed without private subnets, you will not be able to add them after the environment is created" or something like that?